whois-update/src/main/java/net/ripe/db/whois/update/handler/UpdateNotifier.java

```diff
@@ -112,20 +112,15 @@ private boolean notifyOriginAutnum(final PreparedUpdate update) {
                 (object.getType() == ObjectType.ROUTE || object.getType() == ObjectType.ROUTE6));
     }
 
-    private void addNotifications(final Map<CIString, Notification> notifications, final PreparedUpdate update, final UpdateContext updateContext) {
+    public void addNotifications(final Map<CIString, Notification> notifications, final PreparedUpdate update,
+                            final UpdateContext updateContext) {
         final RpslObject object = update.getReferenceObject();
 
         switch (updateContext.getStatus(update)) {
             case SUCCESS:
                 if (updateContext.getAction(update) != Action.NOOP) {
                     addVersionId(update, updateContext);
-                    add(notifications, updateContext, update, Notification.Type.SUCCESS, Collections.singletonList(object), AttributeType.NOTIFY);
-                    add(notifications, updateContext, update, Notification.Type.SUCCESS, rpslObjectDao.getByKeys(ObjectType.MNTNER, object.getValuesForAttribute(AttributeType.MNT_BY)), AttributeType.MNT_NFY);
-                    add(notifications, updateContext, update, Notification.Type.SUCCESS_REFERENCE, rpslObjectDao.getByKeys(ObjectType.ORGANISATION, update.getDifferences(AttributeType.ORG)), AttributeType.REF_NFY);
-                    add(notifications, updateContext, update, Notification.Type.SUCCESS_REFERENCE, rpslObjectDao.getByKeys(ObjectType.IRT, update.getDifferences(AttributeType.MNT_IRT)), AttributeType.IRT_NFY);
-                    if (notifyOriginAutnum(update)) {
-                        add(notifications, updateContext, update, Notification.Type.SUCCESS_REFERENCE, rpslObjectDao.getByKeys(ObjectType.AUT_NUM, update.getDifferences(AttributeType.ORIGIN)), AttributeType.NOTIFY);
-                    }
+                    addNotificationsWithoutVersioning(notifications, update, updateContext, object);
                 }
```

Comment:
This needs to be done for both success and failure

Reply:
should be already working like this, right?
for example when there is an error related to auth we are sending an email to upd-to user